docs/language/core-events.md

```diff
@@ -121,3 +121,51 @@ pub event AccountContractRemoved(
 | `address`   | `Address` | The address of the account the contract gets removed from |
 | `codeHash`  | `[UInt8]` | Hash of the contract source code                          |
 | `contract`  | `String`  | The name of the the contract                              |
+
+### Inbox Value Published
+
+Event that is emitted when a Capability is published from an account.
+
+Event name: `flow.InboxValuePublished`
+
+```cadence
+pub event InboxValuePublished(provider: Address, recipient: Address, name: String, type: Type) 
+```
+
+| Field             | Type      | Description                                  |
+| ----------------- | --------- | -------------------------------------------- |
+| `provider`        | `Address` | The address of the publishing account        |
+| `recipient`       | `Address` | The address of the intended recipient        |
+| `name`            | `String`  | The name associated with the published value |
+| `type`            | `Type`    | The type of the published value              |
+
+To reduce the potential for spam, 
+we recommend that user agents that display events do not display this event as-is to their users, 
+and allow users to restrict whom they see events from. 
+
+### Inbox Value Removed
+
+Event that is emitted when a Capability is removed from an account. 
+
+Event name: `flow.InboxValueRemoved`
```

Comment:
I don't really think there is any benefit to separating these events. In fact, I don't actually know whether the `remover` parameter is even necessary here. If we think about who would actually care about the `Removed` event, it would only be the original publisher and the intended recipient who have any reason to actually listen for this event. Because the value can only be removed if the publisher calls `unpublish` or the recipient calls `claim`, when the value is removed both parties know for certain who was responsible, since they either just submitted a transaction that resulted in the event being removed (in which case it was them), or they didn't (in which case it must have been the other party).